modules/expo-localization/ios/EXLocalization/EXLocalization.m

```diff
@@ -0,0 +1,53 @@
+// Copyright 2018-present 650 Industries. All rights reserved.
+
+#import <EXLocalization/EXLocalization.h>
+
+@implementation EXLocalization
+
+EX_EXPORT_MODULE(ExpoLocalization)
+
++ (BOOL)requiresMainQueueSetup
+{
+  return NO;
+}
+
+- (void)setModuleRegistry:(EXModuleRegistry *)moduleRegistry
+{
+}
+
+- (NSMutableArray *)ensureLocaleTags:(NSArray *)locales
+{
+  NSMutableArray *sanitizedLocales = [NSMutableArray array];
+  
+  for (id locale in locales) {
+    [sanitizedLocales addObject:[locale stringByReplacingOccurrencesOfString:@"_" withString:@"-"]];
+  }
+  
+  return sanitizedLocales;
+}
+
+- (NSDictionary *)constantsToExport
+{
+  BOOL isGreaterThaniOSEight = [[[UIDevice currentDevice] systemVersion] floatValue] >= 9;
```

Comment:
We can safely use `[NSLocale preferredLanguages]` now since https://github.com/expo/expo/commit/1381e35d41e49b23a7de8ade43d71e29497faf39 landed 😉